lib/rules/lint-attribute-indentation.js

```diff
@@ -12,7 +12,7 @@ const canApplyRule = function(node, type, config) {
   let end;
   let start = node.loc.start;
 
-  if (type === 'MustacheStatement') {
+  if (type === 'NonBlockStatement') {
```

Comment:
I'd generally prefer to always pass `node.type` (instead of making our own custom `NonBlockStatement` string here) into this method. We can invert this conditional to keep the simpler logic though.

Something like:

```js
if (type !== 'BlockStatement') {
  // ...snip...
}
```

Reply:
Done: removed `NonBlockStatement` and passed `node.type` directly (see `visitor()`).